tests/test_configuration.py

```diff
@@ -166,8 +177,10 @@ def test_log_level(self):
                 log_level="DeBuG",
             )
 
+    def test_log_level_bad_warning(self):
+
         with self.assertRaises(AssertionError):
-            with self.assertLogs(logger=_logger, level=WARNING):
+            with self.assertLogs(logger=_logger, level=ERROR):
```

Comment:
I might be missing something here... but why is the end result here expecting an error level log, and line 173 only a debug?

Reply:
Fixed, there is no need to call `shutdown` on the tracer provider since we are resetting initial values in `setUp`.